front-end/src/Components/Registration.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react'
+import { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { apiCall } from './Fucntions';
```

Comment:
Try to give some spaces between different types of imports like

```
State1
State2

Page1
Page2
```

etc etc

Reply:
okay I will do so